sdk/src/client/api/high_level.rs

```diff
@@ -133,11 +131,6 @@ impl Client {
             }
         }
 
-        Ok(current_time)
-    }
-
-    // TODO
-    pub async fn get_slot_index(&self) -> Result<SlotIndex> {
-        todo!()
+        Ok(self.get_protocol_parameters().await?.slot_index(current_time))
```

Comment:
Maybe reuse `network_info`? This would do an additional call on wasm

Reply:
Ah, good call